R/deployments.R

```diff
@@ -261,6 +260,54 @@ addToDeploymentHistory <- function(appPath, deploymentRecord) {
   file.rename(newHistory, history)
 }
 
+#' Forget Application Deployment
+#'
+#' Forgets about an application deployment. This is useful if the application
+#' has been deleted on the server, or the local deployment information needs to
+#' be reset.
+#'
+#' @param appPath The path to the content that was deployed, either a directory
+#'   or an individual document.
+#' @param name The name of the content that was deployed (optional)
+#' @param account The name of the account to which the content was deployed
+#'   (optional)
+#' @param server The name of the server to which the content was deployed
+#'   (optional)
+#' @param dryRun Set to TRUE to preview the files/directories to be removed
+#'   instead of actually removing them.
+#' @return NULL, invisibly.
+#'
+#' @details This method removes from disk the file containing deployment
+#'   metadata. If "name", "account", and "server" are all NULL, then all of the
+#'   deployments for the application are forgotten; otherwise, only the
+#'   specified deployment is forgotten.
+#'
+#' @export
+forgetDeployment <- function(appPath = getwd(), name = NULL,
```

Comment:
Is having a default for `appPath` going to be surprising to people? Does this mean a bare call to the function could cause irreversible changes to the filesystem based on implicit arguments?

Reply:
The default of `appPath` to `getwd()` is a de facto standard in this package (most functions have it as the default).

We can add cautionary prompts, though, and I've done so.